integration-test/merge_test.js

```diff
@@ -12,9 +13,9 @@ const TEST_NAMESPACE = 'scratch.merge-test'
 const INVALID_STATEMENT_NAMESPACE = 'scratch.merge-test.invalid-stmt'
 
 const seedObjects = [
-  {hello: 'world'},
-  {foo: 'bar'},
-  {etc: 'and so on'}
+  {id: uuid.v4(), hello: 'world'},
```

Comment:
I wanted to make sure that data from one test wouldn't "contaminate" the results for a subsequent test, since I'm asserting on e.g. the number of objects merged, and if I unthinkingly reuse the same `{foo: "bar"}` object in multiple tests, I might get the wrong result if the node already had it in the store from a previous test. 

I tried doing a garbage collection pass on the datastore after each test, but you have to take the node offline first, and I was getting weird connection reset errors.  So I figured I'd just make sure that all the data objects had a random component, so there's no chance of a duplicate object throwing things off.